# this message is sent only once and contains details about all available parameters.

Header header
ParamInfo[] entries